docs/source/gbd2019_models/risk_effects/low_birthweight_short_gestation/index.rst

```diff
@@ -0,0 +1,353 @@
+birth-death cohort data, which contained three-digit ICD causes of death, but
+also had nearly 30% of deaths coded to causes that are ill-defined, or
+intermediate, in the GBD cause classification system.
+
+Therefore, the GBD modelers analysed the relative risk of all-cause mortality
+across all available sources and selected outcomes based on criteria of
+biological plausibility. **Some causes, most notably congenital birth defects,
+haemoglobinopathies, malaria, and HIV/AIDS, were excluded based on the criteria
+that reverse causality could not be excluded.**
+[GBD-2019-Risk-Factors-Appendix-LBWSG-Risk-Effects]_
+
+.. list-table:: Entities Affected by LBWSG in GBD 2019
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Outcome
+     - Outcome type
+     - Outcome ID
+     - Affected measure
+     - Note
+   * - Diarrheal diseases
+     - Cause
+     - 302
+     - Mortality (GBD YLLs)
+     -
+   * - Lower respiratory infections
+     - Cause
+     - 322
+     - Mortality (GBD YLLs)
+     -
+   * - Upper respiratory infections
+     - Cause
+     - 328
+     - Mortality (GBD YLLs)
+     -
+   * - Otitis media
+     - Cause
+     - 329
+     - Mortality (GBD YLLs)
+     -
+   * - Meningitis
+     - Cause
+     - 332
+     - Mortality (GBD YLLs)
+     -
+   * - Encephalitis
+     - Cause
+     - 337
+     - Mortality (GBD YLLs)
+     -
+   * - Neonatal preterm birth
+     - Cause (PAF-of-1)
+     - 381
+     - Mortality and Morbidity (GBD YLLs and YLDs)
+     - 100% attributable to Low birthweight and short gestation
+   * - Neonatal encephalopathy due to birth asphyxia and trauma
+     - Cause
+     - 382
+     - Mortality (GBD YLLs)
+     -
+   * - Neonatal sepsis and other neonatal infections
+     - Cause
+     - 383
+     - Mortality (GBD YLLs)
+     -
+   * - Hemolytic disease and other neonatal jaundice
+     - Cause
+     - 384
+     - Mortality (GBD YLLs)
+     -
+   * - Other neonatal disorders
+     - Cause
+     - 385
+     - Mortality (GBD YLLs)
+     -
+   * - Sudden infant death syndrome
+     - Cause
+     - 686
+     - Mortality (GBD YLLs)
+     -
+
+Restrictions
+++++++++++++
+
+.. list-table:: Age, Sex, and Outcome Restrictions for LBWSG Relative Risks in GBD 2019
+  :widths: 15 15 20
+  :header-rows: 1
+
+  * - Restriction Type
+    - Value
+    - Notes
+  * - Male only
+    - False
+    -
+  * - Female only
+    - False
+    -
+  * - YLL only
+    - True
+    - Except for Neonatal preterm birth; see :ref:`note <note_on_preterm_birth_DALYs>` below
+  * - YLD only
+    - False
+    -
+  * - Age group start
+    - Early neonatal (0-7 days, age_group_id = 2)
+    -
+  * - Age group end
+    - Late neonatal (7-28 days, age_group_id = 3)
+    - Except for Neonatal preterm birth; see :ref:`note <note_on_preterm_birth_DALYs>` below
+
+.. _note_on_preterm_birth_DALYs:
+
+.. note::
+
+  GBD 2019 attributes 100% of the DALYs due to Neonatal Preterm Birth to the
```

Comment:
Is this different from GBD 2017?

Reply:
No, it's the same as far as I know. Is it confusing that I specify 2019?

Reply:
I edited to just say "GBD" since this is not year-specific.